cmd/bucky/common_sync.go

```diff
@@ -281,6 +294,12 @@ func (ms *metricSyncer) countJobs(jobsd map[string]map[string][]*syncJob) int {
 	return c
 }
 
+func (ms *metricSyncer) metriclog(what string, name string, where string) {
+	if ms.metricErrLogger != nil {
+		ms.metricErrLogger.Printf("%s|%s|%s", what, name, where)
```

Comment:
This is a rather unusual demarcation for a log and could be hard to parse. In the vast majority of cases either ":" or "," is used.

Reply:
`:` is used in date, but ',' is probably better, let's do that.